gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.2-bin.zip
```

Comment:
Shall we test with the latest gradle version?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-socket/pull/2/commits/7830013d54028367951e16e6456910ec8bba1be5